src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -1810,7 +1810,7 @@ namespace VariableSpeedCoils {
                                                RoutineName,
                                                CurrentModuleObject,
                                                state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).Name));
-                        ShowContinueError(state, format("...not found {}=\"{}\".", cAlphaFields(AlfaFieldIncre), AlphArray(16 + (I - 1) * 6)));
+                        ShowContinueError(state, format("...not found {}=\"{}\".", cAlphaFields(AlfaFieldIncre), AlphArray(AlfaFieldIncre)));
```

Comment:
...and this corrects a mistake. These corrections should be described in the PR description. To be explicit, an issue would be created that this PR corrects.

Reply:
Let me create an issue and link it to the PR

Reply:
Created it: [#10137](https://github.com/NREL/EnergyPlus/issues/10137)